### PR TITLE
Add onyomi-hiragana handlebars

### PR DESCRIPTION
**Pull request by rhgg2**
This pull request adds an `onyomi-hiragana` handlebars to the Anki card templates. It does exactly what it says.

I also found a typo in the documentation of the existing onyomi/kunyomi handlebars (hiragana and katakana are inverted).

I have made what I hope are all the necessary changes, except that I have not touched anything in the `test` directory.

### CODE REVIEW

File: ext/data/templates/anki-field-templates-upgrade-v36.handlebars [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+{{/inline}}
+{{=======}}
+{{#*inline "onyomi"}}
+    {{~#each definition.onyomi}}{{.}}{{#unless @last}}, {{/unless}}{{/each~}}
+{{/inline}}
+
+{{#*inline "onyomi-hiragana"}}
+    {{~#each definition.onyomi}}{{hiragana (.)}}{{#unless @last}}, {{/unless}}{{/each~}}
+{{/inline}}
+{{>>>>>>>}}
```
Comment (StefanVukovic99) [inline ext/data/templates/anki-field-templates-upgrade-v36.handlebars:13]:
```suggestion
{{#*inline "onyomi-hiragana"}}
    {{~#each definition.onyomi}}{{hiragana (.)}}{{#unless @last}}, {{/unless}}{{/each~}}
{{/inline}}
```
When adding new handlebars, I don't think we should use the find-and-replace mechanism. Though we did it this way for handlebars updates 35 and 29, I think we should go back to the way it was done in 24 and earlier.

File: ext/settings.html [hunk 2]
```diff
@@ -3088,10 +3088,14 @@ <h1>Yomitan Settings</h1>
                 </tr>
                 <tr>
                     <td><code class="anki-field-marker">{kunyomi}</code></td>
-                    <td>Kunyomi (Japanese reading) for the kanji, expressed as katakana.</td>
+                    <td>Kunyomi (Japanese reading) for the kanji, expressed as hiragana.</td>
```
Comment (StefanVukovic99) [inline ext/settings.html:3091]: This was a mistake in the old descriptions, it was hiragana all along?

**Review by StefanVukovic99 — COMMENTED** [review]
I think you need to `npm run test:unit:write` to update the tests with the new handlebar.

**Review by Kuuuube — CHANGES_REQUESTED** [review]
You need to set the version from 35 to 36 in `test/options-util.test.js` in this area of the file: 

```js
        profileCurrent: 0,
        version: 35,
        global: {
            database: {
                prefixWildcardsSupported: false
            }
        }
```

**Comment by rhgg2** [comment]
Thanks for the feedback - I think everything should be in order now with the tests.

**Review by Kuuuube — APPROVED** [review]

**Pull request merged**